CONTRIBUTING.md

```diff
@@ -19,10 +19,13 @@ give us permission to use your contributions.
    Make a fork of Snapcraft, and create a branch named specifically for the
    feature on which you'd like to work. Make your changes there, adding new
    tests as needed, and make sure the existing tests continue to pass when your
-   changes are complete (for information about running the tests, see the
-   [HACKING][3] document).
+   changes are complete (for information see the [HACKING][3] and [TESTING][4]
+   documents).
 
-3. Squash commits into one, well-formatted commit. If you really feel like there
+3. We try to follow a consistent and readable code style. Read the
```

Comment:
Step 2 discusses making the changes necessary. Should "make your changes while following the style guide" just be part of step 2? Or does it make that step too big?